Author: Michael R. Crusoe <crusoe@debian.org>
Description: drop unused dev dependencies
Forwarded: not-needed

If the next release includes the tests, then some of these should come back
See: https://github.com/onecodex/needletail/issues/81
--- needletail.orig/Cargo.toml
+++ needletail/Cargo.toml
@@ -53,8 +53,6 @@
     "zstd",
 ]
 default = ["compression"]
-python = ["pyo3/extension-module"]
-python_test = ["pyo3"]
 xz2 = ["liblzma"]
 
 [lib]
@@ -95,16 +93,12 @@
 optional = true
 
 [dependencies.liblzma]
-version = "0.3.1"
+version = "0.3"
 optional = true
 
 [dependencies.memchr]
 version = "2.7.2"
 
-[dependencies.pyo3]
-version = "0.21.2"
-optional = true
-
 [dependencies.zstd]
 version = "0.13.2"
 optional = true
@@ -112,21 +106,9 @@
 [dev-dependencies.assert_cmd]
 version = "2"
 
-[dev-dependencies.bio]
-version = "1"
-
-[dev-dependencies.criterion]
-version = "0.5"
-
-[dev-dependencies.escargot]
-version = "0.5.0"
-
 [dev-dependencies.predicates]
 version = "3"
 
-[dev-dependencies.seq_io]
-version = "0.3"
-
 [dev-dependencies.serde]
 version = "1.0"
 
